test/e2e/specs/test.js

```diff
@@ -7,6 +7,29 @@ module.exports = {
     .url('http://localhost:8080')
       .waitForElementVisible('#app', 5000)
       .assert.elementPresent('.randPersonBtn')
+      .pause(5000)
       .end()
+  },
+
+  'startship e2e tests': function (browser) {
+    browser
+        .url('http://localhost:8080')
```

Comment:
Is there a better way to locate the URL? What if the tests need to use a different port?